cmd/common.go

```diff
@@ -39,6 +40,25 @@ const (
 	modeKongEnterprise
 )
 
+type summary struct {
+	Creating int32 `json:"creating"`
+	Updating int32 `json:"updating"`
+	Deleting int32 `json:"deleting"`
+	Total    int32 `json:"total"`
+}
+
+type jsonOutputObject struct {
```

Comment:
Can we move this to the `diff` package? That seems like a more natural fit for this kind of struct. WDYT?

Reply:
Sounds good. Moved.